client/lib/login/index.js

```diff
@@ -80,6 +91,15 @@ export function getSignupUrl(
 		signupUrl = `${ signupUrl }/${ oauth2Flow }?${ oauth2Params.toString() }`;
 	}
 
+	if ( isGravatarOAuth2Client( oauth2Client ) ) {
```

Comment:
Looks like the new two `if` statements can be merged into one since the code inside the blocks are the same.
```suggestion
	if ( isAkismetOAuth2Client( oauth2client ) || isGravatarOAuth2Client( oauth2Client ) ) {
```